shared/schemas/workflow-options-output-schema.json

```diff
@@ -0,0 +1,37 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Workflow Results",
+    "description": "Results of a workflow execution",
+    "type": "object",
+    "properties": {
+        "completedWith": {
```

Comment:
As @batzionb mentioned, we probably don't need this property at all. The workflow developer should set the workflow status to either success or error and so this is redundant.

I would keep the optional `description` property below.

Reply:
After a discussion: keeping/removing the `completedWith` depends on the ability to use `error` field in the instance. Let's keep that decision open/